src/pages/Main/CanvasDiv.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { canvasElement } from "./Canvas";
+import './CanvasDiv.css';
+
+interface CanvasDivProps {
```

Comment:
Let's remove the interface because we aren't using it.

Reply:
done.